test/test_decomp.py

```diff
@@ -433,6 +433,44 @@ def test_uniform(self, device):
         res = torch._decomp.decompositions.uniform(x, low=low, high=high)
         self.assertEqual(ref, res)
 
+    def test_rrelu_with_noise(self, device):
+        # rrelu_with_noise behavior depends on a) whether elements in the input
+        # are <= 0, and b) whether we're in training mode. Cover all cases:
+        dtype = torch.float64
```

Comment:
@eellison So this test works as long as the dtype is float64. If I change to float32 then the test fails for "cpu" (but still succeeds for "cuda")